.github/workflows/turborepo-release-step-3.yml

```
@@ -144,9 +144,13 @@ jobs:
           mv go-artifacts/turbo-go-cross-${{ inputs.release_branch }}/turbo_windows_amd64_v1/bin/* cli/dist-windows-amd64
           chmod a+x cli/dist-windows-amd64/turbo.exe
           chmod a+x cli/dist-windows-amd64/go-turbo.exe
-          mv go-artifacts/turbo-go-cross-${{ inputs.release_branch }}/turbo_windows_arm64/bin/* cli/dist-windows-arm64
-          chmod a+x cli/dist-windows-arm64/turbo.exe
-          chmod a+x cli/dist-windows-arm64/go-turbo.exe
+
+          # rust doesn't have a toolchain for arm + windows + gnu, so we just use the exe from the amd64 build
+          # and rely on windows' arm JITer to do the work for us. this is because the go exe cannot be build w/ msvc
+          mv go-artifacts/turbo-go-cross-${{ inputs.release_branch }}/turbo_windows_amd64_v1/bin/* cli/dist-windows-amd64
+          chmod a+x cli/dist-windows-amd64/turbo.exe
+          chmod a+x cli/dist-windows-amd64/go-turbo.exe
```

If I'm understanding this we want to use the AMD builds for ARM so I think we should be copying to the arm directory and not back over the amd directory. I think we also need to change one of the `mv`s to a copy since the two packages will be sharing the same binary.
```suggestion
          mv go-artifacts/turbo-go-cross-${{ inputs.release_branch }}/turbo_windows_amd64_v1/bin/* cli/dist-windows-arm64
          chmod a+x cli/dist-windows-arm64/turbo.exe
          chmod a+x cli/dist-windows-arm64/go-turbo.exe
```